examples/index.html

```diff
@@ -0,0 +1,537 @@
+<!DOCTYPE html>
```

Comment:
@castastrophe  idk why this file is showing up in my PR, should I go ahead & delete it

Reply:
Yeah, it should be set to gitignore